scripts/build_python.sh

```diff
@@ -105,13 +105,15 @@ source "$CHIP_ROOT/scripts/activate.sh"
 [[ -n "$chip_case_retry_delta" ]] && chip_case_retry_arg="chip_case_retry_delta=$chip_case_retry_delta" || chip_case_retry_arg=""
 
 gn --root="$CHIP_ROOT" gen "$OUTPUT_ROOT" --args="chip_detail_logging=$chip_detail_logging enable_pylib=$enable_pybindings enable_rtti=$enable_pybindings chip_project_config_include_dirs=[\"//config/python\"] $chip_mdns_arg $chip_case_retry_arg"
+echo gn --root="$CHIP_ROOT" gen "$OUTPUT_ROOT" --args="chip_detail_logging=$chip_detail_logging enable_pylib=$enable_pybindings enable_rtti=$enable_pybindings chip_project_config_include_dirs=[\"//config/python\"] $chip_mdns_arg $chip_case_retry_arg"
 
 # Compiles python files
 # Check pybindings was requested
 if [ "$enable_pybindings" == true ]; then
     ninja -C "$OUTPUT_ROOT" pycontroller
 else
     ninja -C "$OUTPUT_ROOT" python
+    echo ninja -C "$OUTPUT_ROOT" python
```

Comment:
same here. Generally wondering if we should move this into build_examples since that one has dry run to show actual used commands (but I explicitly am not asking for this PR  to use build_examples ... that is a reaasonably sized PR by itself)

Reply:
For now, I'll just delete it.